pivotal-api-test/src/test/resources/stories/UpdateStory

```diff
@@ -0,0 +1,19 @@
+Feature: Update a story in project from pivotal tracker
```

Comment:
Notice that the Update and Delete story have the same preconditions (Background)
Probably you can join them in a single feature file.